doc/R-package/Makefile

```diff
@@ -4,6 +4,7 @@ PKGROOT=../../R-package
 # ADD The Markdown to be built here, with suffix md
 discoverYourData.md: $(PKGROOT)/vignettes/discoverYourData.Rmd
 xgboostPresentation.md: $(PKGROOT)/vignettes/xgboostPresentation.Rmd
+xgboostfromJSON.md: $(PKGROOT)/vignettes/xgboostfromJSON.Rmd
 
 # General Rules for build rmarkdowns, need knitr
 %.md:
```

Comment:
Default option for this is to render to .html. The call needs to knitr/rmarkdown be modified to render to .md.

Reply:
Based on the tests so far, the result is md.